content/general/concepts/github/github.md

```diff
@@ -0,0 +1,19 @@
+---
+Title: 'GitHub'
+Description: 'Github is a UI-based version control that uses Git.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'GitHub'
+  - 'Git'
+CatalogContent:
+  - 'learn-git'
+  - 'paths/learn-how-to-build-websites'
+---
+
+GitHub (GH) is a UI-based version control platform that runs on [Git](https://www.codecademy.com/resources/docs/git). It offers many services including version control,  static website hosting, open-source software ([OSS](https://www.codecademy.com/resources/docs/general/open-source)) projects, and much more.
+
+## Github Pages (GH Pages)
```

Comment:
```suggestion
## Github Pages
```

I generally suggest against expanding/collapsing abbreviations in the headers. That should be done in the text.